experimental/packages/sdk-logs/test/common/Logger.test.ts

```diff
@@ -69,22 +61,30 @@ describe('Logger', () => {
       assert.ok(makeOnlySpy.called);
     });
 
-    it('should emit with current Context when includeTraceContext is true', () => {
-      const { logger } = setup({ includeTraceContext: true });
+    it('should emit with current Context', () => {
+      const { logger } = setup({});
       const callSpy = sinon.spy(logger.getActiveLogRecordProcessor(), 'onEmit');
       logger.emit({
         body: 'test log body',
       });
       assert.ok(callSpy.calledWith(sinon.match.any, context.active()));
     });
 
-    it('should emit with empty Context when includeTraceContext is false', () => {
-      const { logger } = setup({ includeTraceContext: false });
+    it('should emit with Context specifie in LogRecord', () => {
```

Comment:
Small typo here, I believe this should be "with Context specified in LogRecord".

Reply:
@JacksonWeber you can suggest fixes like this directly inline so that they can be applied with minimal effort

<img width="218" alt="image" src="https://github.com/open-telemetry/opentelemetry-js/assets/1612643/c469590c-08e6-4eeb-acb0-b90e364d9efb">


```suggestion
    it('should emit with Context specified in LogRecord', () => {
```

Reply:
Looks like this for people with write access to the branch:

<img width="743" alt="image" src="https://github.com/open-telemetry/opentelemetry-js/assets/1612643/d6b8179c-79d5-4374-8b7f-95ea98dc442f">